frontend/src/pages/modelServing/screens/global/InternalServicePopoverContent.tsx

```diff
@@ -20,7 +20,7 @@ const InternalServicePopoverContent: React.FC<InternalServicePopoverContentProps
 
   if (!isInternalServiceEnabled) {
     return (
-      <Stack hasGutter>
+      <Stack>
```

Comment:
If this is the only element and you want to give it some padding, I think this component is what you are looking for https://www.patternfly.org/v4/components/panel#basic

Reply:
Yeah, don't use `Stack` for a single item. What you're getting is a `div`. Just fragment it or `div` wrap it if you need a block element. Arguably using something like `Text` with `p` component prop is all you need here. I think that'll work for what you want (you'll have to verify though).

@DaoDaoNoCode Avoid using Panel when you don't want padding outside -- which `Stack` with or without `hasGutter` (which only does spacing between multiple elements) will do. No need to suggest solutions outside of what is already going on unless you have seen it and it looks bad.

Reply:
Ok, it works as a fragment block, I'm gonna leave it as that.